addons/website_twitter/controllers/main.py

```diff
@@ -11,31 +14,29 @@ def twitter_reload(self):
 
     @http.route(['/get_favorites'], type='json', auth="public", website=True)
     def get_tweets(self, limit=20):
-        key = request.website.twitter_api_key
-        secret = request.website.twitter_api_secret
-        screen_name = request.website.twitter_screen_name
-        cr, uid = request.cr, request.uid
-        debug = request.registry['res.users'].has_group(cr, uid, 'base.group_website_publisher')
-        if not key or not secret:
-            if debug:
+        Key = request.website.twitter_api_key
+        Secret = request.website.twitter_api_secret
+        ScreenName = request.website.twitter_screen_name
+        Debug = request.env['res.users'].has_group('base.group_website_publisher')
```

Comment:
recheck all variable name according to [guideline](https://www.odoo.com/documentation/master/reference/guidelines.html), use camelcase only for model variable